### ISSUE TITLE
Bug of the cloning items

### ISSUE THREAD

**Issue by YerzhanBoss** [VORPCORE/VORP-Hunting#21]
Hello I found cloning items:
You need: 2 people, a bow, a knife and a horse. 
1. Hunt an animal, the more expensive.
2. You peel off the skin and put it on the horse. 
3. Pick up the frame and place it on the horse. 
4. Then sell what you can to the trapper's store. 
5. 2. Two people take the horse by the reins and stand next to it at the butcher's shop at the time of the sale without letting go of the reins
6. The owner of the horse sells the hide several times while the other holds the flood
Bottom line: while the other person holds the horse by the reins, the carcass becomes invisible and is sold indefinitely.

**Comment by AndrewR3K** [comment]
Oh boy, that's an interesting one! 

I'll have to dig further into this one.
One idea for a possible solution could be to disable selling when holding a horse by the reins. 

I'll update this thread if/when I find anything. 😃

**Comment by AndrewR3K** [comment]
@YerzhanBoss Can you let us know if this is still happening for you? The last update had to do with code around this. Curious if that happened to fix it by chance.
